nservicebus/hosting/index.md

```diff
@@ -27,6 +27,8 @@ There are several approaches to hosting.
  * [Endpoint Lifecycle](/nservicebus/lifecycle/)
  * [Critical Error handling](critical-errors.md)
 
+NOTE: Be aware of critical errors, not configuring a critical error callback makes an endpoints unresponsive after a critical error without terminating the process.
```

Comment:
Either way we need to be more explicit in doco, if that's broken by default then shouldn't be custom implementation obligatory?